RS.Fritz.Manager.UI/UserControls/Capture/CaptureControlCaptureViewModel.cs

```diff
@@ -0,0 +1,226 @@
+﻿namespace RS.Fritz.Manager.UI;
+
+using System;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Threading;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.Mvvm.Messaging;
+using Microsoft.Extensions.Logging;
+using RS.Fritz.Manager.API;
+
+internal sealed class CaptureControlCaptureViewModel : FritzServiceViewModel
+{
+    private const int TimerTickIntervalMs = 200;
+    private const string CapturePath = $"/cgi-bin/capture_notimeout";
+    private const string Scheme = "http";
+    private const string Host = "fritz.box";
```

Comment:
This does not work on my network because I don't use the FritzBox as DNS server. Better to use the actual IPv6 or IPv4 address.